internal/txtdiff/diff.go

```diff
@@ -46,7 +46,7 @@ func Diff(t *testing.T, name, got, goldenFile string) {
 
 	gotLines := strings.Split(got, "\n")
 	wantLines := strings.Split(want, "\n")
-	t.Errorf("%s: (-got,+want)\n%s", name, lcsDiff(gotLines, wantLines))
+	t.Errorf("%s: (+got,-want)\n%s", name, lcsDiff(wantLines, gotLines))
```

Comment:
nit: Consider changing either to `(-want,+got)` OR to `lcsDiff(gotLines, wantLines)` so they match

Reply:
The `(+got,-want)` is ordered to match Go's idiomatic order, I'll go with the second.